github-runner-manager/src/github_runner_manager/platform/jobmanager_provider.py

```diff
@@ -213,13 +243,26 @@ def check_job_been_picked_up(self, metadata: RunnerMetadata, job_url: HttpUrl) -
         Returns:
             True if the job has been picked up, False otherwise.
         """
-        configuration = jobmanager_client.Configuration(host=metadata.url)
+        configuration = jobmanager_client.Configuration(host=self._url)
+        #
+        # job_url has the path:
+        # "/v1/job/<job_id>"
```

Comment:
I think this comment should be documented in the caller function.
From the `check_job_been_picked_up` method, what is passed in as `job_url` is transparent to this function.